data/gui/themes/default/window/editor_edit_unit.cfg

```diff
@@ -218,7 +218,8 @@
 
 										[label]
 											definition = "default"
-											label = _ "Level:"
+											#textdomain wesnoth
+											label = _ "Level: "
```

Comment:
I'm against having trailing spaces in strings like this. Even if that currently means that they can't reuse a string, the eventual aim of the cleanup should be to remove the trailing space from the existing translatable strings.

Reply:
Sure

Reply:
corresponding commit dropped.